frontend/src/workbench/LeftSite.tsx

```diff
@@ -11,72 +11,118 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-import { makeStyles } from '@material-ui/core/styles';
-import Tab from '@material-ui/core/Tab';
-import Tabs from '@material-ui/core/Tabs';
+import MuiAccordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
+import MuiCollapse from '@material-ui/core/Collapse';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import { ExplorerWebSocketContainer } from 'explorer/ExplorerWebSocketContainer';
 import React from 'react';
+import { ValidationWebSocketContainer } from 'validation/ValidationWebSocketContainer';
 import { LeftSiteProps } from './LeftSite.types';
 
 const useSiteStyles = makeStyles((theme) => ({
   site: {
-    display: 'flex',
-    flexDirection: 'column',
-    '& > *:nth-child(2)': {
-      flexGrow: 1,
-    },
+    display: 'grid',
+    gridTemplateColumns: 'minmax(0,1fr)',
   },
-  tabsRoot: {
-    minHeight: theme.spacing(4),
-    borderBottomColor: theme.palette.divider,
-    borderBottomWidth: '1px',
-    borderBottomStyle: 'solid',
+}));
+
+const Accordion = withStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'minmax(0,1fr)',
+    gridTemplateRows: 'min-content minmax(0,1fr)',
+    border: '1px solid rgba(0, 0, 0, .125)',
+    boxShadow: 'none',
+    '&:not(:last-child)': {
+      borderBottom: '0px',
+    },
+    '&:before': {
+      display: 'none',
+    },
+    '&$expanded': {
+      margin: '0px',
+    },
   },
-  tabRoot: {
-    minHeight: theme.spacing(4),
-    textTransform: 'none',
+  expanded: {},
+})(MuiAccordion);
+
+const AccordionSummary = withStyles({
+  root: {
+    borderBottom: '1px solid rgba(0, 0, 0, .125)',
+    marginBottom: -1,
+    '&$expanded': {
+      minHeight: 48,
+    },
   },
-  tabLabel: {
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'center',
-    '& > *:nth-child(2)': {
-      marginLeft: theme.spacing(1),
+  content: {
+    '&$expanded': {
+      margin: '0px',
     },
   },
-}));
+  expanded: {},
+})(MuiAccordionSummary);
 
-const a11yProps = (id: string) => {
-  return {
-    id: `simple-tab-${id}`,
-    'aria-controls': `simple-tabpanel-${id}`,
-  };
-};
+const customCollapse = withStyles({
```

Comment:
This should start with an upper case, `CustomCollapse` since this is a component not a regular element :)

Reply:
Done